sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseCreateForm.java

```diff
@@ -260,7 +266,7 @@ protected void addFields() {
 		});
 		facilityOrHome.addValueChangeListener(e -> {
 			FieldHelper.removeItems(facility);
-			if (TypeOfPlace.FACILITY.equals(facilityOrHome.getValue())) {
+			if (TypeOfPlace.FACILITY.equals(facilityOrHome.getValue()) || TypeOfPlace.FACILITY.equals(facilityOrHome.getNullableValue())) {
```

Comment:
This results in an exception when the field is required and HOME is selected, probably because it checks the second condition but can't retrieve a nullable value because the field isn't nullable in the first place

Reply:
Solved... the whole `NullableOptionGroup` had my head hurt anyway. To be honest it does not seem very well implemented, but really that should be handled in another issue as it's used quite a lot.